repositories/index.ts

```diff
@@ -15,6 +16,9 @@ import { LeaderboardRepository } from './LeaderboardRepository';
 import { ResumeRepository } from './ResumeRepository';
 import { UserSocialMediaRepository } from './UserSocialMediaRepository';
 
+const HINT_FOR_RETRIABLE_TRANSACTIONS : string = 'The transaction might succeed if retried.';
+const AMOUNT_OF_RETRIES : number = 10;
```

Comment:
are we consistent with declaring numbers like this at the top of the file or do we generally do this in the Config? small nit either way but for the sake of consistency